guides/common/modules/con_monitoring-hosts-using-red-hat-insights.adoc

```diff
@@ -1,4 +1,23 @@
 [id="Monitoring_Hosts_Using_Red_Hat_Insights_{context}"]
 = Monitoring Hosts Using Red{nbsp}Hat Insights
 
-In this chapter, you can find information about creating host monitoring reports and monitoring your hosts using Red{nbsp}Hat Insights and creating an Insights plan.
+You can use Insights to diagnose systems and downtime related to security exploits, performance degradation, and stability failures.
+You can use the Insights dashboard to quickly identify key risks to stability, security, and performance.
+You can sort by category, view details of the impact and resolution, and then determine what systems are affected.
+
+To use Insights to monitor hosts that you manage with {Project}, you must first install Insights on your hosts and register your hosts with Insights.
+
+For new {Project} hosts, you can install and configure Insights during host registration to {Project}.
+For more information, see xref:Registering_Hosts_by_Using_Global_Registration_{context}[].
+
+For hosts already registered to {Project}, you can install and configure Insights on your hosts by using an Ansible role.
+For more information, see xref:deploying-red-hat-insights-using-the-ansible-role_{context}[].
+
+.Additional Information
+* To view the logs for Insights and all plug-ins, go to `/var/log/foreman/production.log`.
```

Comment:
This statement is a bit unclear: we don't see Insights logs in the `production.log` file. The logs that we can see there are only about the plugin that is responsible for syncing data with Insights.

Reply:
Removed Insights.